src/back/routes/votes.ts

```diff
@@ -66,7 +66,7 @@ export async function updateSnapshotProposalVotes(proposal: ProposalAttributes,
   return votes
 }
 
-export async function getCachedVotes(req: Request) {
+export async function getProposalsCachedVotes(req: Request) {
```

Comment:
Do we need to add "proposals" to the name? Are there any other votes that are not from proposals?